problem_builder/tests/integration/test_mentoring.py

```diff
@@ -436,7 +436,9 @@ def assert_state(answer, mcq, mrq, rating, messages):
         ("feedback_persistence_mcq_no_tips.xml", '.feedback')
     )
     def test_feedback_persistence_tips(self, scenario, tips_selector):
-        # Tests whether feedback is hidden on reload.
```

Comment:
I have a problem with docstrings on ddt methods. Basically it makes things more complicated. 

Reruning only some subsets of tests really hard. DDT generates long method names (like this one: `test_feedback_persistence_tips__O_feedback_persistence_mcq_no_tips_xml__feedback_` --- this is not precise name but close). When this method has a docstring noseruner displays the docstring instead of method name. So I can't just copy that name for debugging purposes.  

Moreover in test logs you'll see couple of tests labeled: "Tests whether feedback (global message and choice tips) is hidden on reload." and only one of them can fail :) 

Long term I'd invest some time on tweaking runner to display method names always, but for now I'd lean towards avoiding docstrings for ddt tests.